modules/ui/fields/combo.js

```diff
@@ -91,7 +91,7 @@ export function uiFieldCombo(field, context) {
         }
 
         if (!field.caseSensitive) {
-            dval = dval.toLowerCase();
+            dval = dval;
```

Comment:
Thanks for looking into this issue. This fix happens to address #10038 as described, but it goes too far: most keys that use this combo field do need to be lowercased. I left a note in https://github.com/openstreetmap/iD/issues/10038#issuecomment-1868331583 about a more correct fix.

Reply:
@1ec5 Thank you for pointing that out, Since this 1line fix requires changes in another repo, 
can I open a PR in id-tagging-schema?